tests/ci/integration/run_mysql_integration.sh

```diff
@@ -88,7 +88,7 @@ main.client_ssl_data_print  : Bug#0002 AWS-LC does not support Stateful session
 main.ssl_cache : Bug#0002 AWS-LC does not support Stateful session resumption (Session Caching).
 main.ssl_cache_tls13 : Bug#0002 AWS-LC does not support Stateful session resumption (Session Caching).
 "> skiplist
-  ./mtr --suite=main --force --parallel=auto --skip-test-list=${MYSQL_BUILD_FOLDER}/mysql-test/skiplist --retry-failure=3 --retry=3 --report-unstable-tests
+  ./mtr --suite=main --force --parallel=$((NUM_CPU_THREADS * 2)) --skip-test-list=${MYSQL_BUILD_FOLDER}/mysql-test/skiplist --retry-failure=3 --retry=3 --report-unstable-tests
```

Comment:
this is already the number available cores, won't doubling it just cause more context switching between processes? is the idea that enough time will be spent waiting on IO that this won't matter?

Reply:
The overall build time for this run is 86 minutes. Logs contain `Using parallel: 16` as expected, the test time took `Spent 18062.931 of 1376 seconds executing testcases` I'll try putting this back to auto and see if anything changes. Looking at this resource utilization CPU usage is still only 15-20% so maybe it was always I/O limited.

![image](https://github.com/aws/aws-lc/assets/41167468/c145eab4-35c9-47c9-9e1f-91e097a18871)